CanonicalLtd/candid
# Update the authentication-required template to match the latest design

## Done
Updated the template to match the [latest desgn](https://user-images.githubusercontent.com/1953388/55154211-21ec2800-514c-11e9-8e1d-a2b2f6a44774.png) by including the latest Vanilla framework into the codebase.

_This is just the authentication-required page. The other pages should look the same_

## QA
- Check out the authentication-required page and see it matches the design
- Check there is a Canonical logo at the top of the page
- Check the favicon loads
- Check there are no HTML validation errors

## Screenshot
![Screenshot_2019-06-18 Candid - Authentication Required](https://user-images.githubusercontent.com/1413534/59699856-30e1f400-91ea-11e9-89b5-dca0dc53ff27.png)

Fixes https://github.com/canonical-web-and-design/rbac-squad/issues/296
